### ISSUE TITLE
Markdown links in fact sets not clickable

### ISSUE THREAD

**Issue by shbhadra** [microsoft/AdaptiveCards#2999]
# Platform
What platform is your issue or question related to? (Delete other platforms).
* Android

# Author or host
*Host

# Version of SDK
* 1.1.1

# Issue
Markdown links inside fact sets are not clickable.
consider the following card json

```
{
  "$schema": "http://adaptivecards.io/schemas/adaptive-card.json",
  "type": "AdaptiveCard",
  "version": "1.0",
  "body": [
 
    {
      "type": "Container",
      "items": [
        {
          "type": "FactSet",
          "facts": [
            {
              "title": "Test Markdown Link :",
              "value": "This is a [Link](https://www.google.com)"
            }
          ]
        }
      ]
    }
  ]
}
```
The text "Link" is rendered looking like a link, however it is not clickable

**Comment by matthidinger** [comment]
@almedina-ms can you take a look at this?

**Comment by jjustinms** [comment]
Do you have an update? @shalinijoshi19

**Comment by shalinijoshi19** [comment]
@almedina-ms please take a look at this with priority after the existing 1.2.1 bugs are done? Thanks!

**Comment by jjustinms** [comment]
@shalinijoshi19 whats the plan to backport to 1.2? 
cc: @shbhadra

**Comment by shalinijoshi19** [comment]
Hi @jjustinms this is already available as of v1.2.1. Thanks
